app/src/main/java/org/sopt/and/screen/signIn/SignInScreen.kt

```diff
@@ -91,32 +109,25 @@ fun SignInScreen(
             Spacer(modifier = Modifier.height(80.dp))
 
             GrayTextField(
-                value = email,
-                placeholderText = "이메일 주소 또는 아이디",
-                onValueChange = { email = it }
+                value = userName,
+                placeholderText = "사용자 이름",
+                onValueChange = { userViewModel.updateUserName(it) }
```

Comment:
```suggestion
                onValueChange = userViewModel::updateUserName
```
이렇게 줄여서 사용해보는 것도 좋을 것 같아요 !